utils.js

```diff
@@ -26,11 +28,13 @@ var toAbsolute = module.exports.toAbsolute = function(fileOrDirectory) {
     return fileOrDirectory;
   }
 
+  var absFileOrDirectory = fileOrDirectory;
+
   if (!path.isAbsolute(fileOrDirectory)) {
-    fileOrDirectory = path.join(process.cwd(), fileOrDirectory);
+    absFileOrDirectory = path.join(process.cwd(), absFileOrDirectory);
```

Comment:
Instead of a new variable, can we `return` here?

```js
if (!path.isAbsolute(fileOrDirectory)) {
  return path.join(process.cwd(), fileOrDirectory);
}

return fileOrDirectory;
```